docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -263,6 +282,17 @@ the Teleport Discovery Service.
 Refer to [Azure SDK Authorization](https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorization)
 for more information.
 
+### Teleport reports no error but VM does not join
+
+Check your Discovery Service config and make sure that the VM you want to
+discover matches. In debug mode, Teleport will log the Subscription IDs and
+names of VMs it discovers.
+
+Unfortunately, the Azure run command API does not report the output of commands,
```

Comment:
```suggestion
The Azure run command API does not report the output of commands,
```
I would just state this as fact. Whether or not this is unfortunate doesn't matter much.